docs/Developers/Newcomers_Guide.rst

```diff
@@ -281,8 +281,6 @@ fork repository.
    In order to avoid the above mentioned cases you can create a new branch
    where you will work on the issue. To do that run:
 
-   ::
```

Comment:
This fix is unrelated to improving emphasis of material. I think it is better to have this in a separate change.